covid_graphs/covid_graphs/country_graph.py

```diff
@@ -80,22 +89,54 @@ def __init__(
         self.log_title = f"Days [since {self.log_xaxis_date_since.strftime('%b %d, %Y')}]"
         self.date_title = "Date"
 
-        max_value = max(
+        self.max_value = max(
             max(self.cropped_cumulative_active),
             max(trace.max_value for trace in self.trace_by_event.values()),
         )
         self.log_yrange = [
             math.log10(max(1, self.cropped_cumulative_active.min())) - 0.3,
-            math.log10(max_value) + 0.3,
+            math.log10(self.max_value) + 0.3,
+        ]
+
+    def _create_slider(self, traces: List[Scatter]):
+        SUBTRACE_COUNT = 4
+
+        last_position = len(self.trace_by_event) - 1
+        # Mark last trace as visible
+        for i in range(last_position * SUBTRACE_COUNT, (last_position + 1) * SUBTRACE_COUNT):
+            traces[i].visible = True
```

Comment:
Nit:
```
for trace in traces[last_position * SUBTRACE_COUNT : (last_position + 1) * SUBTRACE_COUNT]:
    trace.visible = True
```

Reply:
Actually, maybe even this?
```
for trace in traces[-SUBTRACE_COUNT:]:
    trace.visible = True
```

Reply:
Not relevant anymore, filtering based on `legendgroup` now.